mffpy/xml_files.py

```diff
@@ -834,6 +834,10 @@ class Categories(XML):
     _xmlns = r'{http://www.egi.com/categories_mff}'
     _xmlroottag = r'categories'
     _default_filename = 'categories.xml'
+    _type_converter = {
+        'long': int,
+        'person': str,
+    }
```

Comment:
Can you think of more?

Reply:
Looks like we can have type 'fileReference'
```
<key>
                        <keyCode>FILE</keyCode>
                        <data dataType="fileReference">RM271_noise_test_20190501_105754_seg.mff</data>
                    </key>
```